lib/service/service.go

```diff
@@ -3067,15 +3040,18 @@ func (l *proxyListeners) Close() {
 	if !l.db.Empty() {
```

Comment:
is `Empty()` used anywhere else? The close method below is safe to call without the empty check, so I wonder if we can just remove it.

Reply:
It is used in one other spot but yeah the call here is pointless, removed